prow/config.gen.yaml

```diff
@@ -1561,8 +1561,9 @@ presubmits:
           value: /tmp/cache
         - name: GOCACHE
           value: /tmp/cache
-        - name: ISTIO_ENVOY_BASE_URL
-          value: https://storage.googleapis.com/maistra-prow-testing/proxy
+        # FIXME: disabled until we have a proxy build
+        #  - name: ISTIO_ENVOY_BASE_URL
+        #  value: https://storage.googleapis.com/maistra-prow-testing/proxy
```

Comment:
Don't we have a proxy build?

Reply:
We don't have them building on github yet, therefore there are no 1.9 binaries on google storage yet.
I'm talking to Dmitri about this. Meanwhile, we should keep using upstream proxy.

Reply:
Ah. Nevermind. I was thinking dailies.